sycl/test/basic_tests/stdcpp_compat.cpp

```diff
@@ -0,0 +1,7 @@
+// RUN: %clangxx -fsycl -fsyntax-only -Xclang -verify -Xclang -verify-ignore-unexpected=note,warning %s -c -o %t.out
+// RUN: %clangxx -std=c++14 -fsyntax-only -Xclang -verify -I %sycl_include -Xclang -verify-ignore-unexpected=note,warning %s -o -c %t.out
+// RUN: %clangxx -std=c++17 -fsyntax-only -Xclang -verify -I %sycl_include -Xclang -verify-ignore-unexpected=note,warning %s -o -c %t.out
+// RUN: %clangxx -std=c++20 -fsyntax-only -Xclang -verify -I %sycl_include -Xclang -verify-ignore-unexpected=note,warning %s -o -c %t.out
+// expected-no-diagnostics
+
+#include <CL/sycl.hpp>
```

Comment:
There is also similar file: https://github.com/intel/llvm/blob/sycl/sycl/test/warnings/warnings.cpp.
Is it possible and reasonable to merge these files?

Reply:
The tests use similar toolchain  but check different  functionality. We can leave them as different tests.